core/client/src/client.rs

```diff
@@ -126,6 +126,38 @@ pub enum BlockStatus {
 	Unknown,
 }
 
+/// Block import result.
+#[derive(Debug)]
```

Comment:
If you move `execute_and_import_block` into `consensus`, please move this as well.